charts/astronomer/templates/houston/api/houston-backend-secret.yaml

```diff
@@ -14,7 +14,8 @@ metadata:
 type: Opaque
 data:
   {{- if .Values.houston.backendSecretConnection }}
-  connection: {{ (printf "postgresql://%s:%s@%s:%s/%s" .Values.houston.backendConnection.user .Values.houston.backendConnection.pass .Values.houston.backendConnection.host (.Values.houston.backendConnection.port | toString) .Values.houston.backendConnection.db) | b64enc | quote }}
+  # in prisma1 has hardcoded schema, so for now to migrate to prisma2 we need to hardcode it too.
+  connection: {{ (printf "postgresql://%s:%s@%s:%s/%s?schema=houston%24default" .Values.houston.backendConnection.user .Values.houston.backendConnection.pass .Values.houston.backendConnection.host (.Values.houston.backendConnection.port | toString) .Values.houston.backendConnection.db) | b64enc | quote }}
```

Comment:
Does this mean in the future we need a follow up ticket to change the connection here?  Is this only necessary for the migration process?